packages/expo-modules-core/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import './ts-declarations/global';
```

Comment:
feel little concern for this. it looks like having side effect but actually a typing stuff. are there any solutions than this? otherwise, the build/index.js has the `import './ts-declarations/global';` which is not ideal. 

e.g.
```diff
diff --git a/packages/expo-modules-core/src/index.ts b/packages/expo-modules-core/src/index.ts
index fb43f218a9..198b12579f 100644
--- a/packages/expo-modules-core/src/index.ts
+++ b/packages/expo-modules-core/src/index.ts
@@ -1,5 +1,3 @@
-import './ts-declarations/global';
-
 import { DeviceEventEmitter } from 'react-native';
 
 import { EventEmitter, Subscription } from './EventEmitter';
@@ -12,6 +10,8 @@ import { UnavailabilityError } from './errors/UnavailabilityError';
 
 import './sweet/setUpErrorManager.fx';
 
+export type * from './ts-declarations/global';
+
 export { default as uuid } from './uuid';
 
 export {
diff --git a/packages/expo-modules-core/src/ts-declarations/global.ts b/packages/expo-modules-core/src/ts-declarations/global.ts
index 6a37231b55..f70b9ba755 100644
--- a/packages/expo-modules-core/src/ts-declarations/global.ts
+++ b/packages/expo-modules-core/src/ts-declarations/global.ts
@@ -1,6 +1,37 @@
 import type { EventEmitter } from './EventEmitter';
 import type { SharedObject } from './SharedObject';
 
+export interface ExpoGlobal {
+  /**
+   * Host object that is used to access native Expo modules.
+   */
+  modules: Record<string, any>;
+
+  // Natively defined JS classes
+
+  /**
+   * @see EventEmitter
+   */
+  EventEmitter: typeof EventEmitter;
+
+  /**
+   * @see SharedObject
+   */
+  SharedObject: typeof SharedObject;
+
+  // Utils
+
+  /**
+   * Generates a random UUID v4 string.
+   */
+  uuidv4(): string;
+
+  /**
+   * Generates a UUID v5 string representation of the value in the specified namespace.
+   */
+  uuidv5(name: string, namespace: string): string;
+}
+
 /* eslint-disable no-var */
 
 declare global {
@@ -8,34 +39,5 @@ declare global {
    * Global object containing all the native bindings installed by Expo.
    * The object may not be available if Expo modules are not set up correctly in the project.
    */
-  var expo: {
-    /**
-     * Host object that is used to access native Expo modules.
-     */
-    modules: Record<string, any>;
-
-    // Natively defined JS classes
-
-    /**
-     * @see EventEmitter
-     */
-    EventEmitter: typeof EventEmitter;
-
-    /**
-     * @see SharedObject
-     */
-    SharedObject: typeof SharedObject;
-
-    // Utils
-
-    /**
-     * Generates a random UUID v4 string.
-     */
-    uuidv4(): string;
-
-    /**
-     * Generates a UUID v5 string representation of the value in the specified namespace.
-     */
-    uuidv5(name: string, namespace: string): string;
-  };
+  var expo: ExpoGlobal;
 }
```

Reply:
You're right! Thanks for the suggestion, I updated it. Now there are no changes in `build/index.js` and the dev launcher bundles don't have to be updated 👌